roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -97,17 +98,28 @@ class Interface(object):
                 self._iftype = match.group(1)
 
     def down(self):
+        if self.ifname.startswith('tap') and self.iftype() == 'tun':
```

Comment:
We should never call down() on a tun interface, we shouldn't insert this guard here but instead make the decision correctly in `is_neutron_interface()`